ubuntu.md

```diff
@@ -0,0 +1,187 @@
+# Deploying on Ubuntu
+
+Once you have your Ubuntu virtual machine ready, you can deploy your Swift app. This guide assumes you have a fresh install with a non-root user named `swift` with SSH configured. For information on setting this up, check out the platform guides:
+
+- [DigitalOcean](digital-ocean.md)
+
+The [packaging](packaging.md) guide provides an overview of available deployment options. This guide takes you through each deployment option step-by-step for Ubuntu specifically. These examples will deploy SwiftNIO's [example HTTP server](https://github.com/apple/swift-nio/tree/master/Sources/NIOHTTP1Server), but you can test with your own project.
+
+- [Binary Deployment](#binary-deployment)
+- [Source Deployment](#source-deployment)
+
+## Binary Deployment
+
+This section shows you how to build your app locally and deploy just the binary. 
+
+### Build Binaries
+
+The first step is to build your app locally. The easiest way to do this is with Docker. For this example, we'll be deploying SwiftNIO's demo HTTP server. Start by cloning the repository.
+
+```sh
+git clone https://github.com/apple/swift-nio.git
+cd swift-nio
+```
+
+Once inside the project folder, use the following command to build the app though Docker and copy all build arifacts into `.build/install`. Since this example will be deploying to Ubuntu 18.04, the `-bionic` Docker image is used to build.
+
+```sh
+docker run --rm \
+  -v "$PWD:/workspace" \
+  -w /workspace \
+  swift:5.2-bionic  \
+  /bin/bash -cl ' \
+     swift build && \
+     rm -rf .build/install && mkdir -p .build/install && \
+     cp -P .build/debug/NIOHTTP1Server .build/install/ && \
+     cp -P /usr/lib/swift/linux/lib*so* .build/install/'
+```
+
+> Tip: If you are building this project for production, use `swift build -c release`, see [building for production](README.md#building-for-production) for more information.
+
+Notice that Swift's shared libraries are being included. This is important since Swift is not ABI stable on Linux. This means Swift programs must run against the shared libraries they were compiled with. 
+
+After your project is built, use the following command to create an archive for easy transport to the server. 
+
+```sh
+tar cvzf hello-world.tar.gz -C .build/install .
+```
+
+Next, use `scp` to copy the archive to the deploy server's home folder.
+
+```sh
+scp hello-world.tar.gz swift@<server_ip>:~/
+```
+
+Once the copy is complete, login to the deploy server.
+
+```sh
+ssh swift@<server_ip>
+```
+
+Create a new folder to hold the app binaries and decompress the archive.
+
+```sh
+mkdir hello-world
+tar -xvf hello-world.tar.gz -C hello-world
+```
+
+You can now start the executable. Supply the desired IP address and port. Binding to port `80` requires sudo, so we use `8080` instead.
+
+[TODO]: <> (Link to Nginx guide once available for serving on port 80)
```

Comment:
fix?

Reply:
This doesn't show up in the rendered version: https://github.com/swift-server/guides/blob/tn-digital-ocean/ubuntu.md

It is just there to remind me when adding the Nginx guide. I can remove if needed though.